code/utils/composable-subxt/subxt.nix

```diff
@@ -0,0 +1,14 @@
+{ pkgs }:
+pkgs.rustPlatform.buildRustPackage rec {
+  pname = "subxt-cli";
+  # NOTE: keep in sync with Cargo.toml as generator version must be same as dependency to compile generated code
+  version = "0.23.0";
```

Comment:
Is there a way to read this from the Cargo.toml file?

Reply:
no.

Reply:
From local Cargo.toml yes. from remote - now. so I am talking about local - it exists. But not for remote. 
I will not do local in PR as nobody does that, just mention. If somebody will want to try nix.

Reply:
closing for now.